app/javascript/App/components/CoinIndex/index.js

```diff
@@ -1,46 +1,45 @@
 import React, { Component, Fragment } from 'react'
-import {
-  Layout,
-  Card,
-  Button,
-  Menu,
-  Dropdown,
-  Icon,
-  Table,
-  Pagination,
-} from 'antd'
-import styled from 'styled-components'
-import axios from 'axios'
+import { Table } from 'antd'
 import ColumnNames from './ColumnNames'
-
-const { Header, Footer, Content } = Layout
+import CurrencySelector from '../CurrencySelector'
+import SearchCoins from '../shared/SearchCoins'
+import API from '../../lib/localAPI'
 
 class CoinIndex extends Component {
   state = {
+    coins: [],
+    pagination: {
+      defaultPageSize: 10,
```

Comment:
did we change default page size from 100 to 10?

Reply:
Great question - I changed it to 10 to make it easier to test that pagination is working properly and also for performance (prior to the new Coin Search API endpoint) but based on user research they'll likely want to see more per page so happy to bump this back up after showing to internal team members.